datahub-web-react/src/app/ingest/source/builder/RecipeForm/RecipeForm.tsx

```diff
@@ -180,11 +141,7 @@ function RecipeForm(props: Props) {
                     <Collapse.Panel
                         forceRender
                         header={
-                            <SectionHeader
-                                icon={<FilterOutlined />}
-                                text="Filter"
-                                filterSectionTooltip={filterSectionTooltip}
-                            />
+                            <SectionHeader icon={<FilterOutlined />} text="Filter" sectionTooltip={sectionTooltip} />
```

Comment:
then the property of `SectionHeader` can be `sectionTooltip` or even just `tooltip` (since it's implied it's the tooltip for the SectionHeader) but you pass in the tooltip for whatever section you're on. So here you'd pass in `sectionTooltip={filterSectionTooltip}`

Reply:
Done!